tests/system/providers/google/cloud/vertex_ai/example_vertex_ai_auto_ml_forecasting_training.py

```diff
@@ -157,21 +137,15 @@
         project_id=PROJECT_ID,
         trigger_rule=TriggerRule.ALL_DONE,
     )
-    delete_bucket = GCSDeleteBucketOperator(
-        task_id="delete_bucket", bucket_name=FORECAST_GCS_BUCKET_NAME, trigger_rule=TriggerRule.ALL_DONE
-    )
 
     (
         # TEST SETUP
-        create_bucket
-        >> move_dataset_file
-        >> create_forecast_dataset
+        create_forecast_dataset
         # TEST BODY
         >> create_auto_ml_forecasting_training_job
         # TEST TEARDOWN
         >> delete_auto_ml_forecasting_training_job
         >> delete_forecast_dataset
-        >> delete_bucket
```

Comment:
Shouldn't we add `watcher()` to the tests?

Reply:
I agree, bottom imports usually look like this:
```python
    # ### Everything below this line is not part of example ###
    # ### Just for system tests purpose ###
    from tests.system.utils.watcher import watcher

    # This test needs watcher in order to properly mark success/failure
    # when "tearDown" task with trigger rule is part of the DAG
    list(dag.tasks) >> watcher()

from tests.system.utils import get_test_run  # noqa: E402

# Needed to run the example DAG with pytest (see: tests/system/README.md#run_via_pytest)
test_run = get_test_run(dag)
```

Reply:
I have added the watcher to the code